lib/plugins/physics.js

```diff
@@ -142,7 +142,7 @@ function inject (bot, { physicsEnabled }) {
       sendPacketPosition(position, onGround)
     } else if (lookUpdated && bot.isAlive) {
       sendPacketLook(yaw, pitch, onGround)
-    } else if (performance.now() - lastSent.time >= 1000) {
+    } else if (performance.now() - lastSent.time >= 1000 && bot.isAlive) {
```

Comment:
Did you check with a proxy that it is vanilla behavior ? I'm pretty sure this packet need to be sent no matter if the bot is alive.

Reply:
I believe this is vanilla behavior. For the duration that the bot is dead, it does not send a position packet.